src/core/buffer_attributes.h

```diff
@@ -24,6 +24,7 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+#include <iterator>
```

Comment:
Why is this needed?

Reply:
For the `https://www.cplusplus.com/reference/iterator/back_inserter/` used in the buffer_attributes.cc.